### ISSUE TITLE
Make Search Form widget search only products in store?

### ISSUE THREAD

**Issue by jcmatoskx** [elementor/elementor#7170]
Hello! I made my header with Elementor and added a search form to it. However my website is a store, and I wish the search form to perform search only for products in the store like this:
https://evolt.pt/?post_type=product&s=ska
Instead of https://evolt.pt/?s=test

I was told I could do it this way:

This is common in some custom templates. To fix this, just add this code inside the form that contains the input box :

`<input type="hidden" id="search-param" name="post_type" value="product">`

The final form should look something like this:

```
<form class="elementor-search-form" role="search" action="https://evolt.pt" method="get">
<input type="hidden" id="search-param" name="post_type" value="product">
<div class="elementor-search-form__container">
<input placeholder="Procurar um produto..." class="elementor-search-form__input" type="search" name="s" title="Procurar" value="">
<button class="elementor-search-form__submit" type="submit">
<i class="fa fa-search" aria-hidden="true"></i>
<span class="elementor-screen-only">Procurar</span>
</button>
</div>
</form>

```

However how would I edit the elementor search button direcly in order to add this code?

Thank you in advance for the help and advice.

**Comment by bainternet** [comment]
I merged this issue with  #5598, which is on the same subject so we could deal with it more efficiently. 
Please feel free to continue the conversation there.

Thanks for your feedback!

**Comment by danyno** [comment]
try this:

`function custom_list_icon_shortcode($atts, $content = null) {
    $content = do_shortcode($content);
    return '<div style="display: flex; align-items: center; margin: 5px 0;">
                <span style="margin-right: 10px; font-size: 16px; color: #007bff;">&#8593;</span>
                <span style="font-size: 14px; color: #333;">' . $content . '</span>
            </div>';
}
add_shortcode('list_icon', 'custom_list_icon_shortcode');`
